templates/build.html

```diff
@@ -11,24 +11,24 @@
 </main>
 <div class="container">
 
-  <div class="alert alert-info" >
+  <div class="alert alert-info" style="margin-botton:30px;">
     <strong>Check your Email Now</strong>
     <div class="container-fluid">
       <div class="row">
 
         <a class="col-sm-3" target="_blank" href="https://www.gmail.com">
-          <img id='mail'  src="{{ url_for('static',filename='gmail.png') }}">
+          <img id='mail' width="130" style="margin-top:15px;"src="{{ url_for('static',filename='gmail.png') }}">
```

Comment:
Avoid inline CSS

Reply:
> > Currently have my exams going on
> 
> @rahulkumaran rofl me too

Haha, finally done with them! xD

Reply:
@r0mflip Oops! My bad! Will add them to the necessary file!